notebooks/search/03-ELSER.ipynb

```diff
@@ -75,7 +76,13 @@
     "``` -->\n",
     "<!-- # deploy model\n",
     "```json\n",
-    "POST _ml/trained_models/.elser_model_1/deployment/_start?deployment_id=for_search -->"
+    "POST _ml/trained_models/.elser_model_2/deployment/_start?deployment_id=for_search -->\n",
+    "\n",
+    "## Upgrading existing index to `.elser_model_2` ELSER model\n",
+    "\n",
+    "If have already had set up an index with ELSER model `.elser_model_1`, You can easily upgrade to  `.elser_model_2`  model by [Reindexing with ingest pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#reindex-with-an-ingest-pipeline) referencing ELSER `.elser_model_2` and starting querying your data with ELSER right away. \n",
```

Comment:
```suggestion
    "If have already had set up an index with ELSER model `.elser_model_1`, you can easily upgrade to  `.elser_model_2`  model by [Reindexing with ingest pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#reindex-with-an-ingest-pipeline) referencing ELSER `.elser_model_2` and starting querying your data with ELSER right away. \n",
```

Reply:
Simplified the sentence with 

> If you have set up an index with ELSER model .elser_model_1, and would like to upgrade to ELSER v2 model - .elser_model_2, Please follow instructions from the notebook on [how to upgrade an index to use elser model](https://github.com/elastic/elasticsearch-labs/blob/bc36bfa5d1ce56dc1e7412bc8a91db5ba1c36751/notebooks/model-upgrades/upgrading-index-to-use-elser.ipynb)

Commit https://github.com/elastic/elasticsearch-labs/commit/bc36bfa5d1ce56dc1e7412bc8a91db5ba1c36751